pype/hosts/blender/plugin.py

```diff
@@ -20,6 +20,36 @@ def asset_name(
     return name
 
 
+def asset_namespace(
+    asset: str, subset: str
+) -> str:
+    """Return a unique namespace based on the asset name."""
+    avalon_containers = [
+        c for c in bpy.data.collections 
+        if c.name == 'AVALON_CONTAINERS'
+    ]
+    loaded_assets = []
+    for c in avalon_containers:
+        loaded_assets.extend(c.children)
+    if avalon_containers is None:
```

Comment:
You can remove this condition, `avalon_containers` will never be `None`, always will be a list.

Reply:
Oh yeah! I changed the behaviour of the previous part to check if there are other avalon containers published with other assets, and I forgot to change that!